<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationDialog</class>
 <widget class="QDialog" name="ConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="configurationTabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="loginTab">
      <attribute name="title">
       <string>&amp;General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="loginGroupBox">
         <property name="title">
          <string>QRZ Login</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="usernameLabel">
            <property name="text">
             <string>&amp;User Name:</string>
            </property>
            <property name="buddy">
             <cstring>usernameLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="usernameLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="passwordLabel">
            <property name="text">
             <string>&amp;Password:</string>
            </property>
            <property name="buddy">
             <cstring>passwordLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="passwordLineEdit">
            <property name="inputMethodHints">
             <set>Qt::InputMethodHint::ImhHiddenText|Qt::InputMethodHint::ImhNoAutoUppercase|Qt::InputMethodHint::ImhNoPredictiveText|Qt::InputMethodHint::ImhSensitiveData</set>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="stationGroupBox">
         <property name="title">
          <string>Station Info</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="1">
           <widget class="QLineEdit" name="callsignLineEdit"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&amp;Callsign:</string>
            </property>
            <property name="buddy">
             <cstring>callsignLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="gridLabel">
            <property name="text">
             <string>&amp;Grid:</string>
            </property>
            <property name="buddy">
             <cstring>gridLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="gridLineEdit"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="latitudeLineEdit"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="longitudeLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="latitudeLabel">
            <property name="text">
             <string>&amp;Latitude:</string>
            </property>
            <property name="buddy">
             <cstring>latitudeLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="longitudeLabel">
            <property name="text">
             <string>&amp;Longitude:</string>
            </property>
            <property name="buddy">
             <cstring>longitudeLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="stationFillButtonLayout">
            <item>
             <widget class="QPushButton" name="populateFromJs8CallButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Fill from JS8Call</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="populateFromQrzButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Fill from QRZ.com</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="js8CallTab">
      <attribute name="title">
       <string>&amp;JS8Call</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="js8CallGroupBox">
         <property name="title">
          <string>TCP Connection</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="hostnameLabel">
            <property name="text">
             <string>&amp;Hostname:</string>
            </property>
            <property name="buddy">
             <cstring>hostnameLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="hostnameLineEdit">
            <property name="inputMethodHints">
             <set>Qt::InputMethodHint::ImhNone</set>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&amp;Port:</string>
            </property>
            <property name="buddy">
             <cstring>portSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="portSpinBox">
            <property name="maximum">
             <number>65534</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="enableJS8CallCheckBox">
            <property name="text">
             <string>Enable JS8Call Integration</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="configurationDialogButtonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>configurationDialogButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>configurationDialogButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
